pallets/runtime/testnet/src/runtime.rs

```diff
@@ -847,7 +842,6 @@ construct_runtime!(
         StoCapped: sto_capped::{Module, Call, Storage, Event<T>},
         Exemption: exemption::{Module, Call, Storage, Event},
         Settlement: settlement::{Module, Call, Storage, Event<T>, Config},
-        Sto: sto::{Module, Call, Storage, Event<T>},
```

Comment:
This will mess up indices and conflate pallets when decoding storage.
I think we need to keep this line here, but basically make `pallets/sto/src/lib.rs` a skeleton pallet.